webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -502,9 +499,13 @@ def _update_corr_index_dict(
     return updated_index
 
 
-def _layout_figure() -> dict:
```

Comment:
It should be possible to define this closer to the use site by having it be inside `response_correlation_controller`. You  can still access it inside the test by calling
`response_correlation_controller._layout_figure("label")`.